.github/workflows/ci.yml

```diff
@@ -26,3 +26,18 @@ jobs:
       run: earthly --ci +lint
     - name: Run test
       run: earthly --ci +test
+
+  cross:
```

Comment:
These runs couldn't be migrated easily to our Jenkins/gerrit infrastructure, while the Earthly based tests can.

--> I would consider to move them to a separated yaml file `cross.yaml` which can have improved scheduling rule: 
 * every commit
 * every PR
 * every day (!!!)
 
With this approach the cross checks can be executed every day, even if patches are submitted from gerrit (where windows/macos tests are not supported).